<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleStock</class>
 <widget class="QWidget" name="SingleStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="StockPriceHistoryPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="buyButton">
      <property name="text">
       <string>Buy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sellButton">
      <property name="text">
       <string>Sell</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdPrice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdStocks">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Current_Price_Label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stocks_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stocks in Depot:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StockPriceHistoryPlot</class>
   <extends>QWidget</extends>
   <header>stockpricehistoryplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
